include/nbl/video/SPhysicalDeviceFeatures.h

```diff
@@ -330,427 +330,1283 @@ struct SPhysicalDeviceFeatures
+    /* VK_KHR_external_memory */
+
+    // [DEPRECATED] Always Enabled but requires Instance Extensions during API Connection Creation!
+    /* VK_KHR_external_memory_win32 */
+
+    // [DEPRECATED] Always Enabled but requires Instance Extensions during API Connection Creation!
+    /* VK_KHR_external_memory_fd */
+
+    // [TODO LATER] Used for dx11 interop
+    /* VK_KHR_win32_keyed_mutex */
+
+    // [DO NOT EXPOSE] We don't support yet
+    // VK_KHR_external_semaphore_capabilities
+
+    // [DEPRECATED] Promoted to non-optional core VK 1.1
+    /* VK_KHR_external_semaphore */
+
+    // [DEPRECATED] Always Enabled but requires Instance Extensions during API Connection Creation!
+    /* VK_KHR_external_semaphore_win32 */
+
+    // [DEPRECATED] Always Enabled but requires Instance Extensions during API Connection Creation!
+    /* VK_KHR_external_semaphore_fd */
+
+    // [DO NOT EXPOSE] We don't support yet
+    // VK_KHR_push_descriptor
+
+    // [TODO] now we need API to deal with queries and begin/end conditional blocks
+    /* ConditionalRenderingFeaturesEXT *//* VK_EXT_conditional_rendering */
+    bool conditionalRendering = false;
+    bool inheritedConditionalRendering = false;
+
+    // [DEPRECATED] Vulkan 1.2 Core
+    /* VK_KHR_shader_float16_int8 */
+
+    // [DO NOT EXPOSE] We don't support yet
+    // VK_KHR_16bit_storage
+
+    // [DO NOT EXPOSE] this is "swap with damange" known from EGL, cant be arsed to support
+    /* VK_KHR_incremental_present */
+
+    // [TODO] Promoted to VK1.1 non-optional core, haven't updated API to match
+    /* VK_KHR_descriptor_update_template */
+
+    // [DO NOT EXPOSE] We don't support yet
+    // VK_NVX_device_generated_commands
+
+    // [TODO LATER] Don't expose VR features for now
+    /* VK_NV_clip_space_w_scaling */
+
+    // [DO NOT EXPOSE] We don't support yet
+    // VK_EXT_direct_mode_display
+
+    // [DO NOT EXPOSE] We don't support yet
+    // VK_EXT_acquire_xlib_display
+
+    // [DO NOT EXPOSE] We don't support yet
+    // VK_EXT_display_surface_counter
+
+    // [TODO LATER] Requires handling display swapchain stuff
+    /* VK_EXT_display_control */
+
+    // [EXPOSE AS LIMIT]
+    /* VK_GOOGLE_display_timing */
+
+    // [DO NOT EXPOSE] We don't support yet
+    // VK_RESERVED_do_not_use_94
+
+    // [TODO] Investigate
+    /* VK_NV_sample_mask_override_coverage */
+
+    /* VK_NV_geometry_shader_passthrough */
+    // The only reason its not a limit is because it needs geometryShader to be enabled
+    bool geometryShaderPassthrough = false;
+
+    // TODO: look into
+    /* VK_NV_viewport_array2 */
+    /* VK_EXT_image_compression_control */
+    /* VK_EXT_image_compression_control_swapchain */
+    /* VK_EXT_multisampled_render_to_single_sampled probably only useful for stencil-K-routed OIT*/
+    /* VK_EXT_pipeline_properties */
+    /* VK_EXT_external_memory_acquire_unmodified */
+    /* VK_EXT_rasterization_order_attachment_access */
+    /* VK_ARM_shader_core_builtins */
```

Comment:
delete these comments, you have them already.

But make sure to preserve the `VK_EXT_multisampled_render_to_single_sampled` comment